tests/models/beit/test_modeling_beit.py

```diff
@@ -764,19 +764,13 @@ def test_inference_interpolate_pos_encoding(self):
         inputs = processor(images=image, return_tensors="pt", size={"height": 480, "width": 480})
         pixel_values = inputs.pixel_values.to(torch_device)
 
-        # with interpolate_pos_encoding being False an exception should be raised with higher resolution
-        # images than what the model supports.
-        self.assertFalse(processor.do_center_crop)
-        with torch.no_grad():
-            with self.assertRaises(ValueError, msg="doesn't match model"):
-                model(pixel_values, interpolate_pos_encoding=False)
-
```

Comment:
We always interpolate, error raising was removed for ZoeDepth in
https://github.com/huggingface/transformers/pull/30136/files#diff-3f84bebd6be8d9c0f5c5068199f5c49eac8489d5fa466fb6fa08b0365e78dba4